R/build_lm.R

```diff
@@ -578,7 +619,7 @@ glance.lm_exploratory <- function(x, pretty.name = FALSE, ...) { #TODO: add test
 
 #' special version of glance.lm function to use with build_lm.fast.
 #' @export
-glance.glm_exploratory <- function(x, pretty.name = FALSE, ...) { #TODO: add test
+glance.glm_exploratory <- function(x, pretty.name = FALSE, threshold = 0.5, ...) { #TODO: add test
```

Comment:
Thought the parameter name threshold is lacking context. Something like binary_classification_threshold might give context better.